server/neptune/workflows/internal/activities/vcs.go

```diff
@@ -0,0 +1,22 @@
+package activities
+
+import (
+	"context"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/github"
+)
+
+// TODO: Initial implementation will support GH VCS, but we will need to eventually abstract to other platforms
```

Comment:
we probably don't need to worry about abstracting for a long time so it's fine to focus on gh.